index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/b08094b2a2.js" crossorigin="anonymous"></script>
+    <title>Personal Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav><i class="fas fa-bars"></i></nav>
+
+        <h2 class="header">Hey there.
+            I’m Ian.
+            <font class="text_color"> I’m a software developer</font>
+        </h2>
```

Comment:
- [ ] You have done a great job to ensure that the heading uses `hx` tag. However, this is the main title of the page and therefore, I suggest you use the `h1` tag here, It helps the search engine identify this as the main website title. ⚡
- [ ] In case you feel more secure, you may consider having `Hey there. I'm Ian` in `h1` tag and `I'm a software developer` in `h2` tag to maintain the HTML element hierarchy ☺️.